lib/reel/session.rb

```diff
@@ -21,21 +21,26 @@ def self.store
       @store ||= Celluloid::Extras::Hash.new
     end
 
+    # will be storing all timers for deleting Session values
+    def self.timers_hash
+      @timers ||= {}
+    end
+
+    # changing/modifying configuration
+    def self.configuration options={}
+      if @options
+        @options.merge! options if options.is_a? Hash
+      else
+        @options = DEFAULT_CONFIG.merge options if options.is_a? Hash
+      end
+      @options
+    end
+
     # This module will be mixed in into Reel::Request
     module RequestMixin
       include Celluloid::Internals::Logger
       include Reel::Session::Crypto
 
-      # changing/modifying configuration
-      def configuration options={}
-        if @options
-         @options.merge! options if options.is_a? Hash
-       else
-         @options = DEFAULT_CONFIG.merge options if options.is_a? Hash
-       end
-       @options
-      end
-
       # initializing session
       def initialize_session
         @bag = Store.new self
```

Comment:
with the `include Celluloid` in store.rb below, this will cause a new actor/thread per request